### PR TITLE
Rename AlwaysOn to Azure Mission-Critical

### PR DESCRIPTION
**Pull request by heoelri**
Following the PR https://github.com/Azure/Mission-Critical/pull/12 renaming AlwaysOn to Azure Mission-Critical. This PR updates all links to the main [Mission-Critical](https://github.com/Azure/Mission-Critical) GitHub repo. 

It does not (yet) update and replace all occurences of AlwaysOn - including folders, directories, application components.

### CODE REVIEW

File: CONTRIBUTE.md [hunk 1]
```diff
@@ -1,22 +1,22 @@
-# How to Contribute to AlwaysOn
+# How to Contribute to Azure Mission-Critical
 
 ## Content
 
-The structure of the AlwaysOn repository is broken down into three overarching directories:
+The structure of the Azure Mission-Critical repository is broken down into three overarching directories:
 
-* `/docs/` contains the majority of AlwaysOn documentation, covering the architectural framework and design approach as well as detailed documentation to accompany the reference implementation.
+* `/docs/` contains the majority of Mission-Critical documentation, covering the architectural framework and design approach as well as detailed documentation to accompany the reference implementation.
```
Comment (heoelri) [inline CONTRIBUTE.md:7]:
```suggestion
* `/docs/` contains the majority of Azure Mission-Critical documentation, covering the architectural framework and design approach as well as detailed documentation to accompany the reference implementation.
```

File: CONTRIBUTE.md [hunk 2]
```diff
@@ -1,22 +1,22 @@
...
 * `/src/` contains all source code and technical artifacts for the reference implementation along with low level implementation documentation.
 * `/.ado/pipelines` contains the Azure DevOps pipelines to build and deploy the core reference implementation.
 
 ## Content Changes and Pull Requests
 
-To add or edit content within the AlwaysOn repository, please take a fork of the repository to iterate on changes before subsequently opening a Pull Request (PR) to get your forked branch merged into the main branch for the AlwaysOn repository. Your PR will be reviewed by the core engineering team for the AlwaysOn project, and once approved, your content accessible to everybody.
+To add or edit content within the Mission-Critical repository, please take a fork of the repository to iterate on changes before subsequently opening a Pull Request (PR) to get your forked branch merged into the main branch for the Mission-Critical repository. Your PR will be reviewed by the core engineering team for the Azure Mission-Critical project, and once approved, your content accessible to everybody.
```
Comment (heoelri) [inline CONTRIBUTE.md:13]:
```suggestion
To add or edit content within the Azure Mission-Critical repository, please take a fork of the repository to iterate on changes before subsequently opening a Pull Request (PR) to get your forked branch merged into the main branch for the Mission-Critical repository. Your PR will be reviewed by the core engineering team for the Azure Mission-Critical project, and once approved, your content accessible to everybody.
```

**Comment by hjscherer** [comment]
> > rather
> 
> Thinking the same - especially the RIs are hard. Maybe we should start with the README.md first and do the rest later. @hjscherer?

Agreed, I would focus on .md content and we can do code later if needed. Lets review this and then get it merged.

File: docs/reference-implementation/Getting-Started.md [hunk 3]
```diff
@@ -28,7 +28,7 @@ This guide offers two paths: Using Azure DevOps Portal or script-based via Azure
 The process to deploy AlwaysOn is comprised of the following steps:
 
 1) Create an [Azure DevOps organization and project](#create-a-new-azure-devops-project)
-1) Generate your own repository based on the [AlwaysOn GitHub template](https://github.com/Azure/AlwaysOn-Foundational-Online/generate) repository
+1) Generate your own repository based on the [AlwaysOn GitHub template](https://github.com/Azure/Mission-Critical-Foundational-Online/generate) repository
```
Comment (hjscherer) [inline docs/reference-implementation/Getting-Started.md:31]: should just be "Mission-Critical-online"

File: docs/reference-implementation/Networking-Design-Decisions.md [hunk 4]
```diff
@@ -41,7 +41,7 @@ The default version of the Reference Implementation of AlwaysOn does not use [fu
 
 These decisions are explained further below:
 
-> It is acknowledged that these decisions might not suit every use case, for instance in some regulated industries. Therefore, there is an alternative version of the Reference Implementation  which deploys in a [Private Mode](https://github.com/Azure/AlwaysOn-foundational-private). However, this comes potentially at the expense of higher cost and reliability risk. Thus, the requirements and impact should be fully understood before making the switch.
+> It is acknowledged that these decisions might not suit every use case, for instance in some regulated industries. Therefore, there is an alternative version of the Reference Implementation  which deploys in a [Private Mode](https://github.com/Azure/Mission-Critical-foundational-private). However, this comes potentially at the expense of higher cost and reliability risk. Thus, the requirements and impact should be fully understood before making the switch.
```
Comment (hjscherer) [inline docs/reference-implementation/Networking-Design-Decisions.md:44]: Need to be "Mission-Critical-online"

File: docs/reference-implementation/Networking-Design-Decisions.md [hunk 5]
```diff
@@ -55,7 +55,7 @@ These decisions are explained further below:
 
 - The Reference Implementation uses Private Endpoints to access all PaaS instead of relying on Service Endpoints only. This has two reasons:
   - Due to some limitations in the way the infrastructure gets deployed through Terraform, Service Endpoints could not be used for all services, so Private Endpoints would have been required at least partially in any case.
-  - The [Private Mode of AlwaysOn](https://github.com/Azure/AlwaysOn-foundational-private) requires the use of Private Endpoints for all used services. So using them also for the default, public mode, brings consistency and simplification in the deployment.
+  - The [Private Mode of AlwaysOn](https://github.com/Azure/Mission-Critical-foundational-private) requires the use of Private Endpoints for all used services. So using them also for the default, public mode, brings consistency and simplification in the deployment.
```
Comment (hjscherer) [inline docs/reference-implementation/Networking-Design-Decisions.md:58]: same here. See above

File: docs/reference-implementation/Networking-Design-Decisions.md [hunk 6]
```diff
@@ -64,7 +64,7 @@ As described above, to remove the public endpoint on the compute clusters, anoth
 
 A more significant change if using private endpoints is the switch from hosted Build Agents (managed by Microsoft) to [self-hosted agents](https://docs.microsoft.com/azure/devops/pipelines/agents/agents?view=azure-devops&tabs=browser#install) which will need to be VNet-integrated in order to reach private services like Key Vault or AKS. Managing these agents and keeping them updated adds additional overhead and is not recommended as long as there is no actual requirement to switch to a fully private deployment.
 
-To deploy Reference Implementation in a private configuration, follow the guides of [this GitHub repository](https://github.com/Azure/AlwaysOn-foundational-private).
+To deploy Reference Implementation in a private configuration, follow the guides of [this GitHub repository](https://github.com/Azure/Mission-Critical-foundational-private).
```
Comment (hjscherer) [inline docs/reference-implementation/Networking-Design-Decisions.md:67]: Mission-Critical-online

File: src/app/AlwaysOn.UI/src/components/MainPage.vue [hunk 7]
```diff
@@ -3,9 +3,9 @@
    <h1>😎 Welcome,</h1>
    <p>... to Azure AlwaysOn! This is a reference implementation of a community driven project designed to demo and document the process, requirements and design decisions to setup a highly scalable and available application in Microsoft Azure.</p>
 
-   <a href="https://github.com/Azure/AlwaysOn"><img src="/img/logo.png" title="AlwaysOn logo" style="width: 50%" /></a>
+   <a href="https://github.com/Azure/Mission-Critical"><img src="/img/logo.png" title="AlwaysOn logo" style="width: 50%" /></a>
 
-   <p>You can find more details <a href="https://github.com/Azure/AlwaysOn-Foundational-Online">on GitHub</a>.</p>
+   <p>You can find more details <a href="https://github.com/Azure/Mission-Critical-Foundational-Online">on GitHub</a>.</p>
```
Comment (hjscherer) [inline src/app/AlwaysOn.UI/src/components/MainPage.vue:8]: remove "foundational"

File: src/infra/workload/README.md [hunk 8]
```diff
@@ -14,7 +14,7 @@ The reference implementation can be used to deploy to different flavors of the A
 - A "public" version which does not fully lock down all services, but in turn it can be deployed using Azure DevOps-hosted Build Agents. Plus, developers and administrators can more easily connect to the resources and debug them.
 - A fully "private" version which locks all traffic to the services down to Private Endpoints. This provides even tighter security but requires the use of self-hosted, VNet-integrated Build Agents. Also, for any debugging etc. users most connect through Azure Bastion and Jump Servers.
 
-Head over to [this GitHub repository](https://github.com/Azure/AlwaysOn-foundational-private) for detailed instructions how to set up the private version.
+Head over to [this GitHub repository](https://github.com/Azure/Mission-Critical-foundational-private) for detailed instructions how to set up the private version.
```
Comment (hjscherer) [inline src/infra/workload/README.md:17]: Here the same - see above

**Review by hjscherer — CHANGES_REQUESTED** [review]

File: docs/reference-implementation/Getting-Started.md (hunk 3 above)
Comment (heoelri) [reply docs/reference-implementation/Getting-Started.md]: fixed

**Comment by heoelri** [comment]
@hjscherer updated the icon as well - i think this is PR is now ready for a review.

File: README.md [hunk 9]
```diff
@@ -1,5 +1,5 @@
-[![Azure Mission-Critical Application](./icon-light.png#gh-light-mode-only "Azure Mission-Critical Online")](./README.md)
-[![Azure Mission-Critical Application](./icon-dark.png#gh-dark-mode-only "Azure Mission-Critical Online")](./README.md)
+![Azure Mission-Critical Application](./icon-light.png#gh-light-mode-only "Azure Mission-Critical Online")
+![Azure Mission-Critical Application](./icon-dark.png#gh-dark-mode-only "Azure Mission-Critical Online")
```
[the remark this answers is not in the document]
Comment (hjscherer) [reply README.md]: looks like it doesn't work with a description

**Review by hjscherer — APPROVED** [review]

File: README.md (hunk 9 above)
[the remark this answers is not in the document]
Comment (heoelri) [reply README.md]: have you used the one w/o "online" now? 

image

**Review by CalCof — APPROVED** [review]

**Pull request merged**